modules/comment-likes/comment-like-count.js

```diff
@@ -24,7 +24,7 @@ jQuery( document ).ready( function( $ ) {
 	function fetchCounts() {
 		var batchRequest = {
 			path: '/batch',
-			data: 'urls[]=' + APIqueue.join( '&urls[]=' ),
+			data: 'urls[]=' + APIqueue.map( 'encodeURIComponent' ).join( '&urls[]=' ),
```

Comment:
Sorry, but you need `map( encodeURIComponent )` here instead of `map( 'encodeURIComponent' )`.